src/raftstore/store/snap.rs

```diff
@@ -2493,7 +2493,7 @@ pub mod tests {
             .unwrap();
 
             // TODO: this size may change in different RocksDB version.
-            let snap_size = 1342;
+            let snap_size = 1438;
```

Comment:
better use const instead of a magic number? and how to calculate it?

Reply:
It's depends on rocksdb's sst format. I think it is ok in test.